### PR TITLE
Laboratory Work #1, Khodyreva Darya - 20FPL2

### PR DESCRIPTION
**Pull request by DaryaKhodyreva**

### CODE REVIEW

File: pipeline.py [hunk 1]
```diff
@@ -92,8 +93,11 @@ def validate_dataset(path_to_validate):
 
 
 def main():
```
Comment (dmitry-uraev) [inline pipeline.py:95]: all starting code should be written under the If __main__ == "__name__": line

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]

File: pipeline.py (hunk 1 above)
Comment (DaryaKhodyreva) [reply pipeline.py:95]: I've understood, will fix it

**Comment by DaryaKhodyreva** [comment]
I have some difficulties with the last "git push". How can I fix a problem with "parser checks"?

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
I see you have troubles with CI - try pushing to GitHub some changes and see if problem is solved. Make sure you have green PR on Friday assignment.

**Pull request closed without merging**